src/store/__mocks__/callZome.js

```diff
@@ -1,15 +1,33 @@
-export const callZome = async (_, __, zomeName, fnName, payload) => {
-  console.log('calling mock callZome with', zomeName, fnName, payload)
-
+export const callZome = async (_, rootState, zomeName, fnName, payload) => {
+  console.log(`calling mock callZome with ${zomeName}.${fnName}() with payload : `, payload)
   switch (fnName) {
+    case 'agent_stats':
+      return {
+        active: 1,
+        agents: 1
+      }
     case 'create_channel':
+      /* eslint-disable no-case-declarations */
+      const { name, ...channelEntry } = payload
```

Comment:
It does, and in fact the .entry key is actually the only info left on the channel obj after .name key is removed, which is why I named it as such.  But... I have no problem making it just channel (plus it might be a bit more future compatible). :)